File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        res=""
        if(len(strs)==1):
            return strs[0]
        for i in range(0,len(strs[0])):
            c=0
            
            for j in range(1,len(strs)):
                if(strs[0][0:i+1]==strs[j][0:i+1]):
                    c+=1
            if(c==(len(strs)-1)):
                res=strs[0][0:i+1]
        return res      

File: 0014-longest-common-prefix/0014-longest-common-prefix.js
/**
 * @param {string[]} strs
 * @return {string}
 */
var longestCommonPrefix = function(strs) {
//         res=""
//         if(strs.length==1){
//             return strs[0]
//             }
//         for (i=0;i<strs.length+1;i++){
//             c=0
            
//             for(j=1;j<strs.length;j++){
//                 if(strs[0].slice(0,i+2)==strs[j].slice(0,i+2)){
//                     c+=1}

//             }
//             if(c==(strs.length-1)){
//                 res=strs[0].slice(0,i+2)
//                 } 
                
//             }
//   return res       
let res = "";
    if (strs.length === 1) {
        return strs[0];
    }
    for (let i = 0; i < strs[0].length; i++) {
        let c = 0;

        for (let j = 1; j < strs.length; j++) {
            if (strs[0].substring(0, i + 1) === strs[j].substring(0, i + 1)) {
                c++;
            }
        }
        if (c === (strs.length - 1)) {
            res = strs[0].substring(0, i + 1);
        }
    }
    return res;     
} 

   